src/Grpc/JsonTranscoding/src/Microsoft.AspNetCore.Grpc.JsonTranscoding/Internal/DescriptorRegistry.cs

```diff
@@ -45,29 +57,28 @@ private void AddFileDescriptorsRecursive(FileDescriptor fileDescriptor)
         }
     }
 
-    private void AddNestedEnumDescriptorsRecursive(MessageDescriptor messageDescriptor)
+    private void AddDescriptorsRecursive(MessageDescriptor messageDescriptor)
```

Comment:
As someone with no context, I would find "AddMessageDescriptorsRecursive" clearer (assuming it's accurate).

Reply:
This method is adding message descriptors and enum descriptors nested inside messages. I think the current name is better.